google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/DownloadJob.java

```diff
@@ -39,8 +41,8 @@ private DownloadJob(
     this.parallelDownloadConfig = parallelDownloadConfig;
   }
 
-  public @NonNull List<ApiFuture<DownloadResult>> getDownloadResults() {
-    return downloadResults;
+  public @NonNull List<DownloadResult> getDownloadResults() {
```

Comment:
When we add javadocs, we will want to be sure to mention these methods are blocking.